package.json

```diff
@@ -58,7 +55,7 @@
     "async-each": "^1.0.1",
     "extend": "^3.0.2",
     "google-auth-library": "^5.5.0",
-    "google-gax": "^1.14.2",
+    "google-gax": "^2.0.0",
```

Comment:
Will this break due to Node 8 deprecation efforts?

Reply:
Upgrade gax to fix one linter issue.
@alexander-fenster What do you think? I am not very sure.

Reply:
We're going to throw this in as part of the 2.x library release, so I think it's fine to leave it now.

Reply:
👍